docs/releases/2.2.1.txt

```diff
@@ -25,3 +25,7 @@ Bugfixes
 
 * Fixed a regression in Django 2.2 that caused a single instance fast-delete
   to not set the primary key to ``None`` (:ticket:`30330`).
+
+* Fixed the recreation of check constraints (and possibly other operations)
```

Comment:
I would add a note about partial indexes, e.g.:
`of check constraints (and possibly other operations)` -> `of check constraints and partial indexes`